docs/designers-developers/designers/animation.md

```diff
@@ -0,0 +1,33 @@
+# Animation
```

Comment:
@nosolosw @mkaz @chrisvanpatten not sure how to add this to the handbook/manifest with the new setup. Feel free to push a commit to add it.

Reply:
I don't think I'll be able to push a commit myself today :( But @mkaz can help I bet? The tl;dr is that you just need to add an entry in docs/toc.json and run the docs build and the manifest will be updated from that file.

Reply:
Looks like it was added, let me know if I can help